test/e2e/copy_from_image_test.go

```diff
@@ -261,3 +335,14 @@ func TestCopyErrorsWhenCopyToTarAndGenerateOutputLockFile(t *testing.T) {
 		t.Fatalf("expected copy to fail when --lock-output is provided with a tar destination, got %v", err)
 	}
 }
+
+func stopNetworkAccessToSourceRegistry(t *testing.T, env *Env) {
```

Comment:
are we assuming here that $IMGPKG_E2E_AIRGAPPED_TEST_REPO is somehow related to `docker stop registry-for-airgapped-testing`? it seems that if we going to take on docker dependency that it should be container to the test instead of splitting it over test-e2e.sh and go test.